features/router.js

```diff
@@ -1,9 +1,18 @@
 const express = require("express");
+const path = require("path");
 const router = express.Router();
 
 router.use("/", require("./projects/projectRouter"));
 router.use("/", require("./test/testRouter"));
 
+router.get("/404", (req, res) => {
+    res.sendFile(path.join(__dirname, "/errors/404.html"));
+});
+
+router.get("/500", (req, res) => {
+    res.sendFile(path.join(__dirname, "/errors/500.html"));
+});
+
 router.get("/", (req, res) => {
     res.redirect("/projects");
 });
```

Comment:
You can change the single quotes setting in the .prettierrc to true and let the double quotes be replaced by single ones (on this file)

Reply:
merge with master was done, now prettier changes to single quote